Topics/Tech_Stacks/Iframe.md

```diff
@@ -0,0 +1,25 @@
+# Learning Iframe
+
+## Introduction
+
+An Iframe is a html element that can let programmer to display a child website inside a parent website. It is supported by many browsers, like chrome, firefox and safari.
+
+## Why use Iframe
+Sometimes we want to let user to interact with a webpage without open a new webpage tage and redict to it. For example, we want to embed a survey inside a webpage so that the user can conveniently fill out the survey after reading the previous content in this webpage.
```

Comment:
Please check for typos and grammatical errors.

"Sometimes we want to let **user to** interact with a webpage **without open a new webpage** **tage** and **redict** to it. For example, we want to embed a survey inside a webpage so that the user can conveniently fill out the survey after reading the previous **content** in this webpage."